src/components/FilePreview/components/FileRenderer/hooks.test.js

```diff
@@ -1,58 +1,60 @@
-import { mockUseKeyedState } from '@edx/react-unit-test-utils';
-
-import { useRenderData, stateKeys } from './hooks';
+import React from 'react';
+import { useRenderData } from './hooks';
 import { errorStatuses, errorMessages } from '../constants';
 
-const state = mockUseKeyedState(stateKeys);
-
 describe('useRenderData', () => {
   const props = {
     file: { fileName: 'file.pdf', fileUrl: 'http://example.com' },
     formatMessage: jest.fn(),
   };
+  let setStateSpy;
+  const setValue = jest.fn();
+
   beforeEach(() => {
-    jest.clearAllMocks();
-    state.mock();
+    setStateSpy = jest.spyOn(React, 'useState').mockImplementation((value) => [value, setValue]);
   });
+
   afterEach(() => {
-    state.resetVals();
+    setStateSpy.mockRestore();
+    jest.clearAllMocks();
   });
 
   it('start with initial state', () => {
     useRenderData(props);
-    state.expectInitializedWith(stateKeys.errorStatus, null);
-    state.expectInitializedWith(stateKeys.isLoading, true);
+    expect(setStateSpy.mock.calls.length).toBe(2);
```

Comment:
It'd be good to also verify we're calling it with the correct params here.

```suggestion
    expect(setStateSpy.mock.calls.length).toBe(2);
    expect(setStateSpy).toHaveBeenCalledWith(null);
    expect(setStateSpy).toHaveBeenCalledWith(true);
```

we do lose a bit of clarity as to *why* `null` and `true` are the params we're looking for. I really like how you handled it in `pdfHooks.test.js` where you had an `initialState` object to use so it is very clear why the values are being checked for.

Maybe for this and the one in `useAssessmentData.test.js‎` we could just use comments to add that context

```js
    expect(setStateSpy.mock.calls.length).toBe(2);
    expect(setStateSpy).toHaveBeenCalledWith(null); // errorStatus
    expect(setStateSpy).toHaveBeenCalledWith(true); // isLoading
```